site/app/templates/polls/PollForm.twig

```diff
@@ -416,7 +435,7 @@
     $(document).ready(function() {
         setEventHandlers();
         $("#new-poll-form").on("change click", submitErrorChecks);
-
+        initializePollType();
```

Comment:
Is there a reason why this can't be "initialized" on the server side by adding the appropriate class/CSS style?  Doing it on the client side leads to a flash as content moves around after the initial page load.

Reply:
fixed